R/epi_workflow.R

```diff
@@ -113,14 +155,12 @@ predict.epi_workflow <- function(object, new_data, ...) {
         i = "Do you need to call `fit()`?"))
   }
   components <- list()
-  the_fit <- workflows::extract_fit_parsnip(object)
-  the_recipe <- workflows::extract_recipe(object)
   components$mold <- workflows::extract_mold(object)
   components$forged <- hardhat::forge(new_data,
                                       blueprint = components$mold$blueprint)
   components$keys <- grab_forged_keys(components$forged,
                                       components$mold, new_data)
-  components <- apply_frosting(object, components, the_fit, the_recipe, ...)
+  components <- apply_frosting(object, components, new_data, ...)
```

Comment:
Don't we still need `the_fit` and `the_recipe`? Am I missing something in the changes?

Reply:
The `epi_workflow` (object) contains the recipe and fit, so it seemed unnecessary to include them separately.

Reply:
We might need some more tests then. I would think this would fail for sure.

For the future, when PRs touch lots of files like this one, it's helpful for me if your initial comment goes through the logic of the things you do. That way I can figure out where to look, and if there are simplifications that happen across multiple files, it's easier for me to understand the idea.